src/java.base/share/classes/com/sun/crypto/provider/CipherCore.java

```diff
@@ -871,18 +871,17 @@ private byte[] prepareInputBuffer(byte[] input, int inputOffset,
         // calculate total input length
         int len = Math.addExact(buffered, inputLen);
         // calculate padding length
-        int totalLen = len;
```

Comment:
Well, personally, I'd prefer to replace `len` with `totalLen`, less changes overall and better naming matching the comment on line 871.

Reply:
Okay.